test/features/MiniBannerPreselect.ts

```diff
@@ -12,7 +12,7 @@ const expectShowsFullPageWhenPreselectIsClicked = async ( wrapper: VueWrapper<an
 const expectPreselectsAmountWhenPreselectIsClicked = async ( wrapper: VueWrapper<any> ): Promise<any> => {
 	await wrapper.find( '.wmde-banner-mini-button-preselect' ).trigger( 'click' );
 
-	expect( wrapper.find<HTMLInputElement>( '.wmde-banner-select-group-input:checked' ).element.value ).toStrictEqual( '5' );
+	expect( wrapper.find<HTMLInputElement>( '.wmde-banner-select-group-input:checked' ).element.value ).toStrictEqual( '10' );
```

Comment:
Oh, here's where my banner testing pattern hits limitations. I'll try think of a way we can pass in a value to this in the banner test.